README.md

```diff
@@ -904,7 +904,7 @@ print(log)
 
 ### Create PixReversals
 
-You can reverse a PixRequest either partially or totally using a PixReversal.
+Pix chargebacks can be created when fraud is detected on a transaction or a system malfunction
```

Comment:
Pix chargebacks can be created when a fraud or a system malfunction is detected on a transaction.